src/module.c

```diff
@@ -8659,6 +8737,43 @@ int RM_SetModuleUserACL(RedisModuleUser *user, const char* acl) {
     return ACLSetUser(user->user, acl, -1);
 }
 
+sds addACLsToUser(user *u, sds username, sds *argv, int argc);
+
+RedisModuleString * RM_SetModuleUserACLString(RedisModuleCtx * ctx, RedisModuleUser *user, const char* acl) {
+    sds sacl = sdsnew(acl);
+    int argc;
+    sds *argv = sdssplitargs(sacl, &argc);
+
+    sds error = addACLsToUser(user->user, NULL, argv, argc);
```

Comment:
```suggestion
    sds error = addACLsToUser(user->user, user->name, argv, argc);
```

Reply:
I'm not sure about this.  as name is for when we create the user via this mechanism, unsure thats valid from an rm function

Reply:
is `user->user == NULL` is valid? if yes - what are your expectations in this case?

Reply:
I don't see how user->user can be NULL.  if user isn't null, user->user should be a redis internal user.